extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -158,6 +162,9 @@ export const portMustBeNumber = localize('portMustNotBeNumber', "Port must a be
 export const valueCannotBeEmpty = localize('valueCannotBeEmpty', "Value cannot be empty");
 export const dockerImageLabelPrefix = 'source=sqldbproject';
 export const dockerImageNamePrefix = 'sqldbproject';
+export const sqlServer2019 = 'SQL Server 2019';
```

Comment:
can the SqlTargetPlatform be reused instead of having separate consts for these 3 platforms?